### PR TITLE
feat(rootfolder): add support for getting the root folder id

### PR DESCRIPTION
**Pull request by dominique-pfister**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/helix-onedrive-support",
-  "version": "1.2.1",
+  "version": "1.3.1",
```
Comment (tripodsan) [inline package-lock.json]: why did you change this? ideally, if the deps don't change, don't change the package-lock.

File: src/OneDrive.js [hunk 2]
```diff
@@ -269,6 +269,19 @@ class OneDrive extends EventEmitter {
     }
   }
 
+  /**
+   */
+  async getRootFolderId(driveId) {
```
Comment (tripodsan) [inline src/OneDrive.js]: this returns a drive item, right? so why is it called `getRootFolderId` ?

better: `getDriveRootItem` ?

**Comment by tripodsan** [comment]
please also update the API in `OneDrive.d.ts`

File: package-lock.json (hunk 1 above)
Comment (dominique-pfister) [reply package-lock.json]: This is the version of helix-onedrive-support itself, and it didn’t match the one in package.json - shouldn’t they?

File: src/OneDrive.js (hunk 2 above)
Comment (dominique-pfister) [reply src/OneDrive.js]: Right, I changed the method signature without changing its name, will do!
